unified-codes/tools/scripts/dgraph/migration/v2/DataParser.ts

```diff
@@ -177,6 +185,8 @@ export class CSVParser extends DataParser {
               const key = column
                 .trim()
                 .toLowerCase()
+                .replace(/[\r\n]/g, '')
+                .replace(/(uc8) *\(([^)]*)\)/, '$1 $2')
```

Comment:
Whoo, more regexes! A nice-to-have, but a potential justification for a helper class for spreadsheets, e.g. `column.getKey()` 😃 .

Reply:
also noted in #311 so there were some constants added in that one.